doc-content/enterprise-only/openshift/githooks-proc.adoc

```diff
@@ -0,0 +1,80 @@
+For example, a simple git hooks directory can provide a post-commit hook that pushes the changes upstream. If the project was imported into {CENTRAL} from a repository, this repository remains configured as the upstream repository. Create a file named `post-commit` with permission values `755` and the following content:
++
+[subs="attributes,verbatim,macros"]
+----
+git push
+----
++
+. Supply the Git hooks directory to the {CENTRAL} deployment. You can use a configuration map or a persistent volume.
+.. If the Git hooks consist of one or several fixed script files, use a configuration map. Complete the following steps:
+... Change into the Git hooks directory that you have created.
+... Create an OpenShift configuration map from the files in the directory. Run the following command:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap git_hooks --from-file=<file_1>=<file_1> --from-file=<file_2>=<file_2> ...
+----
++
+Where `file_1`, `file_2` and so on are git hook script files. For example:
++
+[subs="attributes,verbatim,macros"]
+----
+oc create configmap git_hooks --from-file=post-commit=post-commit
+----
++
+... Mount the configuration map on the {CENTRAL} deploymentm in the path that you have configured:
++
+[subs="attributes,verbatim,macros"]
+----
+oc set volume dc/<myapp>-{PRODUCT_INIT}centr --add --type configmap --configmap-name git_hooks  --mount-path=<git_hooks_dir> --name=git_hooks
+----
++
+Where `<myapp>` is the application name that was set when configuring the template and `<git_hooks_dir>` is the value of `GIT_HOOKS_DIR` that was set when configuring the template.
++
+.. If the Git hooks consist of many files or require writing any files at runtime, use a persistence volume. You must create a persistent volume, create a persistent volume claim and associate the volume with the claim, transfer files to the volume, and mount the volume in the `_myapp_-{PRODUCT_INIT}centr` deployment config (where _myapp_ is the application name). For instructions about creating and mounting persistence volumes, see https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/html/developer_guide/dev-guide-persistent-volumes[Using persistent volumes]. For instructions about copying files onto a persistent volume, see https://blog.openshift.com/transferring-files-in-and-out-of-containers-in-openshift-part-3/[Transferring files in and out of containers].
++
+. Wait a few minutes, then review the list and status of pods in yor project. Because {CENTRAL} does not start until you provide the Git hooks directory, the {KIE_SERVER} might not start at all. To see if it has started, check the output of the following command:
++
+[subs="attributes,verbatim,macros"]
+----
+oc get pods
+----
++
+If a working {KIE_SERVER} pod is not present, start it:
++
+[subs="attributes,verbatim,macros"]
+----
+oc rollout latest dc/<myapp>-kieserver
+----
++
+Where `<myapp>` is the application name that was set when configuring the template.
```

Comment:
As before for the sentence frag.
This is a complicated procedure, but I don't mind this level of complexity. I think it's OK.